lib/experimental/editor-settings.php

```diff
@@ -43,12 +43,28 @@ function gutenberg_enable_experiments() {
 
 /**
  * Sets a global JS variable used to trigger the availability of form & input blocks.
+ *
+ * @deprecated 19.0.0 Use gutenberg_enable_block_experiments().
  */
 function gutenberg_enable_form_input_blocks() {
+	_deprecated_function( __FUNCTION__, 'Gutenberg 19.0.0', 'gutenberg_enable_block_experiments' );
```

Comment:
Given that we're folding the form input blocks into the block experiments, is the [related settings field](https://github.com/WordPress/gutenberg/blob/e829dd2e797eea003df4bd553aff64ae7f9c733d/lib/experiments-page.php#L94) still necessary?

Reply:
I don't think this function itself would be called by anything other than the Gutenberg plugin. How about removing this function?

Reply:
> Given that we're folding the form input blocks into the block experiments, is the [related settings field](https://github.com/WordPress/gutenberg/blob/e829dd2e797eea003df4bd553aff64ae7f9c733d/lib/experiments-page.php#L94) still necessary?

I don't know the history or status of the form blocks experiment, so I want keep it separate for now and avoid needing to migrate the settings.

> I don't think this function itself would be called by anything other than the Gutenberg plugin. How about removing this function?

Good point, though I prefer to be cautious for now. I'll set a reminder to come back and remove it in a couple of releases.